spec/active_record/events_spec.rb

```diff
@@ -116,5 +116,22 @@
       task.update_columns(expired_at: nil)
       expect(Task.not_expired).to include(task)
     end
+
+    describe 'for date fields' do
+      it 'consider today\'s event over' do
```

Comment:
Let's use double quotes here (and below) to avoid escaping characters.

Reply:
ah, of course :)